table/sst_file_writer.cc

```diff
@@ -253,10 +254,11 @@ Status SstFileWriter::Open(const std::string& file_path) {
   TableBuilderOptions table_builder_options(
       r->ioptions, r->mutable_cf_options, r->internal_comparator,
       &int_tbl_prop_collector_factories, compression_type, compression_opts,
-      r->skip_filters, cf_id, r->column_family_name, unknown_level,
-      0 /* creation_time */, 0 /* oldest_key_time */,
-      0 /* file_creation_time */, "SST Writer" /* db_id */, db_session_id,
-      0 /* target_file_size */);
+      cf_id, r->column_family_name, unknown_level, false /* is_bottommost */,
+      TableFileCreationReason::kMisc, 0 /* creation_time */,
+      0 /* oldest_key_time */, 0 /* file_creation_time */,
+      "SST Writer" /* db_id */, db_session_id, 0 /* target_file_size */);
+  table_builder_options.skip_filters = r->skip_filters;
```

Comment:
Please add a comment why you are setting skip_filters here even tho you hope to deprecate it (for historical/cleanup purposes later)/